handbook/company/pricing-features-table.yml

```diff
@@ -830,9 +805,9 @@
   tier: Free
   jamfProHasFeature: no
   jamfProtectHasFeature: no
-- industryName: Custom device data for help desk
+- industryName: Device inventory
```

Comment:
Consolidated `Custom device data for help desk` and `Targeted device scoping` and renamed it **“Device inventory”** for clarity. Added information about Teams and Labels to the description/tooltip.

Reply:
@nonpunctual, Just realized that the features we consolidated are from different pricing tiers. **"Custom device data for help desk"** is a free feature. **"Targeting device scoping"** is premium. How do you want to handle this? Keep separate, but rename **"Custom device data for help desk"** to **"Device inventory"**?

Reply:
Reverted this consolidation with this commit https://github.com/fleetdm/fleet/pull/20609/commits/e8f8ed9d643f8e8c7903bd18d29159005d0deafb